frame/dvm/evm/precompiles/transfer/src/kton.rs

```diff
@@ -108,6 +108,9 @@ impl<T: Config> Kton<T> {
 					}
 				}
 
+				<darwinia_evm::Pallet<T>>::deposit_event(darwinia_evm::Event::TransferToWKTON(
+					caller, value,
+				));
```

Comment:
I think it's better to add this event in contract.

Reply:
I have discussed with Echo and XueLei about this solution. Something note:
1. All contract events are wrapped by `Evm::Log(...)` which is hard to catch by application developers, as the Subql can only filter event by module, and cannot filter contract by address. It will cause index overhead for searching transfer precompile related events in Evm::logs.
2. Once WKTON is deployed, it is hard to upgrade.

Reply:
We can catch and filter event in contract by thegraph, not `Evm::Log(...)` by subquery. WKTON is an application, it does not belong to EVM pallet.

Reply:
@sxlwar Can you explain in more detail why you are unable to use thegraph in your case?

Reply:
It will be used on the subql side, not thegraph. Subql  can parse substrate events only ,not thegraph events